Docs/developer_guide/build_instructions/macos.md

```diff
@@ -5,74 +5,108 @@
 The prerequisites listed below are required to be able to configure/build/package/test Slicer.
 
 - XCode command line tools must be installed:
-```
+
+```bash
 xcode-select --install
 ```
+
 - A CMake version that meets at least the minimum required CMake version [here](https://github.com/Slicer/Slicer/blob/master/CMakeLists.txt#L1)
-- Large File Storage for git is required. (`brew install git-lvs`)
+- Large File Storage for git is required. (`brew install git-lfs`)
 - Qt 5: **tested and recommended**.
   - For building Slicer: download and execute [qt-unified-mac-x64-online.dmg](https://download.qt.io/official_releases/online_installers/qt-unified-mac-x64-online.dmg), install Qt 5.15, make sure to select `qtscript` and `qtwebengine` components.
   - For packaging and redistributing Slicer: build Qt using [qt-easy-build](https://github.com/jcfr/qt-easy-build#readme)
-- Setting `CMAKE_OSX_DEPLOYMENT_TARGET` CMake variable specifies the minimum macOS version a generated installer may target.  So it should be equal to or less than the version of SDK you are building on. Note that the SDK version is set using `CMAKE_OSX_SYSROOT` CMake variable automatically initialized during CMake configuration.
+- Setting `CMAKE_OSX_DEPLOYMENT_TARGET` CMake variable specifies the minimum macOS version a generated installer may target. So it should be equal to or less than the version of SDK you are building on. Note that the SDK version is set using `CMAKE_OSX_SYSROOT` CMake variable automatically initialized during CMake configuration.
 
 ## Checkout Slicer source files
 
 Notes:
+
 - While it is not enforced, we strongly recommend you to *avoid* the use of *spaces* for both the `source directory` and the `build directory`.
-- Due to maximum path length limitations during build the build process, source and build folders must be located in a folder with very short total path length. This is expecially critical on Windows and MacOS. For example, `/sq5` has been confirmed to work on MacOS.
+- Due to maximum path length limitations during build the build process, source and build folders must be located in a folder with very short total path length. This is expecially critical on Windows and MacOS. For example, `/opt/s` has been confirmed to work on MacOS.
```

Comment:
On a recent version of macOs, is it an issue to use '/opt' ? see https://apple.stackexchange.com/questions/399421/opt-folder-no-longer-accessible-in-finder-on-catalina

Reply:
I was able to add it to the dock on the side of the finder as a shortcut, so that hasn't been a problem.

We should point out if we haven't that you need to use `sudo` for `mkdir` and `chown` in `/opt`

Reply:
@pieper , do you have anything apart from slicer build folders in your `/opt`? 

I've read the linked stackexchange post and took another look in my `/opt`. On my machine it looks like `/opt` was brought in by migration assistant from an older laptop. When I was setting up the Slicer build I didn't have problems writing to `/opt` because the folder was already there and is contained files that were created by me like 6 years ago, before the actual computer was assembled on the factory.
On the other hand when I tried to create the `/sq5` folder the system didn't let me (even with `sudo`) because `/` "is" the mentioned read-only system volume. 

It might be a case what @jcfr is talking about.
In order to test this someone with a mac that had 10.15+ pre-installed on the new machine and who didn't use migration assistant from the older mac to check if `/opt` is available and writable with or without `sudo`

Reply:
On my old machine I used `/s`.  When I moved to my new mac manually (no migration assistant) and started using `/opt` because it was the shortest path that already existed and I could write to (with sudo).  There's also a `/otp/cisco` that's part of the hospital vpn and it is owned by root so it was created by the installer.  So I think it's a reasonable place to use.  I also put an `/opt/tmp` and '/opt/data` for stuff I want local but don't want in my home directory (don't really care if it gets lost).

Reply:
@pieper and your new mac came with Catalina, right?

Reply:
> @pieper and your new mac came with Catalina, right?

yes